detection-rules/attachment_html_smuggling_with_base64_script_source.yml

```diff
@@ -0,0 +1,59 @@
+name: "Attachment: HTML Smuggling with base64 script source"
+description: |
+  Potential HTML smuggling attacks from new senders.
+  This rule detects HTML files as attachments or within archives that contain base64 script sources from new or outlier senders. 
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    any(attachments,
+        (
+          .file_extension in~ ("html", "htm", "shtml", "dhtml")
+          or .file_type == "html"
+          or .content_type == "text/html"
+        )
+        and any(file.explode(.),
+                length(.scan.strings.strings) < 20
+                and any(.scan.strings.strings,
+                        regex.icontains(., 'script src="data:text/html;base64')
+                )
+        )
+    )
```

Comment:
We also have `file.parse_html` where you can search the raw text. Doesn't help for the archive case unfortunately. Once that's out, we can use that here to make it simpler.

Reply:
Sweeet thanks for the reminder, I think this came out before that was ready. I'll check it out!